### PR TITLE
tests: remove ubuntu 21.10 from spread tests due to end of life

### PR DESCRIPTION
**Pull request by sergiocazzolato**
the end of life is 07-14  so we remove the test support in spread executions.

### CODE REVIEW

File: tests/main/lxd-mount-units/task.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ details: |
     are correct for snapfuse.
 
 # only 20.04+, we want lxd images that come with snaps preinstalled.
-systems: [ubuntu-20*, ubuntu-21.10*]
+systems: [ubuntu-20*]
```
Comment (pedronis) [inline tests/main/lxd-mount-units/task.yaml:8]: shouldn't this test also run on 22?

File: tests/main/lxd-try/task.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 summary: Check that try command works inside lxd container
 
-systems: [ubuntu-20.04-*, ubuntu-21.10-*]
+systems: [ubuntu-20.04-*]
```
Comment (pedronis) [inline tests/main/lxd-try/task.yaml:3]: same question

**Review by pedronis — COMMENTED** [review]
question

**Comment by mvo5** [comment]
> @mvo5 Hi, with this change we will need to remove ubuntu 21.10 as required in github

I removed it now from the required list.

File: tests/main/lxd-mount-units/task.yaml (hunk 1 above)
Comment (mvo5) [reply tests/main/lxd-mount-units/task.yaml]: This got fixed with 11951

**Review by mardy — APPROVED** [review]
LGTM,  thanks!

**Pull request merged**